app/assets/components/ui-elements/oj-annotation.html

```diff
@@ -1,27 +1,41 @@
 <polymer-element name='oj-annotation' attributes="annotation color paper">
   <template>
     <div class='container' style=" background-color: {{color}}" >
-      <div class='content'>
-        <p>{{annotation.body}}</p>
-      </div>
 
-      <div class='comments'>
-        <template repeat="{{response in annotation.responses}}">
-          <p>{{response.body}}</p>
+      <div  on-tap="{{toggleCollapsed}}" class='content'>
+        <template if="{{collapsed}}">
+          <p layout horizontal>
+            <span> {{annotation.body | truncate}}</span>
+            <span flex></span>
+            <span>{{annotation.responses | count}} replies</span>
```

Comment:
Not sure but I don't think so because the observer model watches changes on the object in the expression which would be a function in the case of annotation.responses.length. That doesnt change. We want this to be recomputed when annotation.responses changes. hence using the count helper. I might be wrong but this works and i think its pretty readable and obvious so I am happy to leave it as is just now